### ISSUE TITLE
Error in method _save_records()

### ISSUE THREAD

**Issue by xMAC94x** [observerss/pygodaddy#2]
Hello Jingchao Hu,
I hope you read this bug issue :)
I am using your pygodaddy Libary for 1 Year and it worked perfectly fine till last week.

I've done some research and it looks like godaddy rejects the HTTP POST in the _save_records() method.
- Logging in works fine
- find_dns_records() lists all my A Records, so it works fine too.
- _add_record seems to work
- _save_records returns False, The Answer from Godaddly is:
```python
 "<?xml version="1.0" encoding="utf-8"?><string xmlns="http://tempuri.org/">FAILURE|Invalid Request.</string>"
```

It looks like the Protocol has changed.
I used the following code for the last 2 methods:
```python
 result_up1 = client._add_record(prefix="foo", value="112.123.123.123", index=3, record_type="A")
 result_up2 = client._save_records(domain="example.com", index=3, record_type="A")
```
result_up1 was true, and result_up2 was false.

I Googled ad bit and found this: "Godaddy CSRF protection". They fixed the Error. Maybe this is causing a Problem here? But i am not really sure, i am just guessing. 

Thanks for this great Libary !

**Comment by DanChianucci** [comment]
My forked branch contains a fix for this issue.  you were correct in that it was the csrf fix that caused this

**Comment by xMAC94x** [comment]
fixed with Pull request #3
